### ISSUE TITLE
Cypress Tests fail since Version 2

### ISSUE THREAD

**Issue by baijii** [2nthony/vue-remix-icons#9]
Possibly not a problem with `vue-remix-icons` or I am holding it wrong. But for sure useful for others.

I updated `vue-remix-icons` from `1.1.1` to `2.0.3`. After that, in my Cypress tests, the first test will timeout as it tries to load all icon components at once (which takes about a minute at my local machine). Downgrading to `1.1.1` again fixed it for me.

I then looked, if I had a default import like `import icons from 'vue-remix-icons'`, but none found.

My environment:
- Node 16.17.0
- Vite 3.0.9
- Vue 3.2.37
- Cypress 10.6.0

**Comment by 2nthony** [comment]
Thanks for the feedback. is this project using typescript?

**Comment by baijii** [comment]
Yes, it does. Typescript 4.7.4 (the current version of yesterday)
